servers/src/main/java/tachyon/master/next/filesystem/meta/InodeTree.java

```diff
@@ -220,7 +220,7 @@ public void deleteInode(Inode inode) throws FileDoesNotExistException {
 
     mInodes.remove(inode);
     mPinnedInodeFileIds.remove(inode.getId());
-    inode.reverseId();
+    inode.delete();
```

Comment:
I agree that this is cleaner.

O ok. Are you saying that there is nowhere in the code that actually checks for "id < 0"? If nothing uses it, then we could just get rid of the flag altogether. (this could be a TODO)